smart-contracts/test/Unlock/initializers.js

```diff
@@ -3,6 +3,9 @@ const unlockContract = artifacts.require('Unlock.sol')
 const { reverts } = require('truffle-assertions')
 const getProxy = require('../helpers/proxy')
 
+const TRUFFLE_VM_ERROR =
+  'VM Exception while processing transaction: reverted with reason string'
```

Comment:
I wonder if this would be worth moving to some utils/constants file?

Reply:
yep I added a `helpers/constants.js` with the needed error messages